 <div style="text-align: center; background-color: #D4AF37; font-family: 'Trebuchet MS', Arial, sans-serif; color: white; padding: 20px; font-size: 40px; font-weight: bold; border-radius: 10px 10px 0 0; box-shadow: 0px 6px 8px rgba(0, 0, 0, 0.2);">Adidas Sales
</div>
<p style="text-align: center;">



## <b><span style='color:#df2c14'> </span><span style='color:#ff6242'> Import Libraries & Load Data </span></b>

In [1]:
#Import the Libraries
import numpy as np
import pandas as pd

import seaborn as sns
import matplotlib.pyplot as plt
import plotly.graph_objects as go
import plotly.express as px

from warnings import filterwarnings
filterwarnings('ignore')

In [2]:
df = pd.read_excel('Adidas US Sales Datasets.xlsx')


In [3]:
table= pd.DataFrame({'unique':df.nunique(),
                   'Null':df.isnull().sum(),
                   'Nullpercent':df.isnull().sum()/len(df)*100,
                   'type':df.dtypes, "duplicated":df.duplicated().sum()})
table

unique  Null  Nullpercent            type  duplicated
Retailer               6     0          0.0          object           0
Retailer ID            4     0          0.0           int64           0
Invoice Date         724     0          0.0  datetime64[ns]           0
Region                 5     0          0.0          object           0
State                 50     0          0.0          object           0
City                  52     0          0.0          object           0
Product                6     0          0.0          object           0
Price per Unit       146     0          0.0         float64           0
Units Sold           361     0          0.0           int64           0
Total Sales         3512     0          0.0         float64           0
Operating Profit    6510     0          0.0         float64           0
Operating Margin     127     0          0.0         float64           0
Sales Method           3     0          0.0          object           0

In [4]:
df.head(5)

Retailer  Retailer ID Invoice Date     Region     State      City  \
0  Foot Locker      1185732   2020-01-01  Northeast  New York  New York   
1  Foot Locker      1185732   2020-01-02  Northeast  New York  New York   
2  Foot Locker      1185732   2020-01-03  Northeast  New York  New York   
3  Foot Locker      1185732   2020-01-04  Northeast  New York  New York   
4  Foot Locker      1185732   2020-01-05  Northeast  New York  New York   

                     Product  Price per Unit  Units Sold  Total Sales  \
0      Men's Street Footwear            50.0        1200     600000.0   
1    Men's Athletic Footwear            50.0        1000     500000.0   
2    Women's Street Footwear            40.0        1000     400000.0   
3  Women's Athletic Footwear            45.0         850     382500.0   
4              Men's Apparel            60.0         900     540000.0   

   Operating Profit  Operating Margin Sales Method  
0          300000.0              0.50     In-store  
1          150000.0              0.30     In-store  
2          140000.0              0.35     In-store  
3          133875.0              0.35     In-store  
4          162000.0              0.30     In-store

<div style="position: relative; background-color: #EAEAEA; font-size: 20px; font-family: Georgia; border: 3px solid #FF5733; padding: 10px; margin: 10px 0; color: #333;">
    <div style="position: absolute; top: -0px; left: 40%; transform: translateX(-50%); background-color: #FF5733; padding: 5px 15px; font-weight: bold; font-size: 24px; color: white;">Questions</div>

 1. Sales Performance Analysis: Which products are excelling in sales, and which are underperforming?
 2. Regional Market Analysis: Which stores are experiencing strong sales, and which ones are lagging?
 3. Efficiency of Sales Methods: Which sales method is more effective — in-store or online?
 5. Price Optimization: Is there a specific price range that achieves better sales than others?
 6. Product Portfolio Optimization I: Determine which products are most profitable, segmented by location.
 7. Market Expansion Opportunities: Assess the best and worst performing stores based on their locations.

   
  

## <b><span style='color:#df2c14'> </span><span style='color:#ff6242'>Sales Performance Analysis: Which products are excelling in sales, and which are underperforming?</span></b>

In [12]:
Product_Sales = df.groupby(by='Product')['Total Sales'].sum().reset_index(name='Total_Sales').sort_values(by='Total_Sales', ascending=False)

In [13]:
Product_Sales

Product  Total_Sales
2      Men's Street Footwear  208826244.0
3            Women's Apparel  179038860.0
1    Men's Athletic Footwear  153673680.0
5    Women's Street Footwear  128002813.0
0              Men's Apparel  123728632.0
4  Women's Athletic Footwear  106631896.0

Sales analysis tells us that last 3 products may be under performing, however we will check units sold and operating profit to see what the data tells us

In [15]:
Product_Sold = df.groupby(by='Product')['Units Sold'].sum().reset_index(name='Units_Sold').sort_values(by='Units_Sold', ascending=False)

In [16]:
Product_Sold

Product  Units_Sold
2      Men's Street Footwear      593320
1    Men's Athletic Footwear      435526
3            Women's Apparel      433827
5    Women's Street Footwear      392269
4  Women's Athletic Footwear      317236
0              Men's Apparel      306683

In [17]:
Product_OP = df.groupby(by='Product')['Operating Profit'].sum().reset_index(name='Operating Profit').sort_values(by='Operating Profit', ascending=False)

In [18]:
Product_OP

Product  Operating Profit
2      Men's Street Footwear       82802260.62
3            Women's Apparel       68650970.56
1    Men's Athletic Footwear       51846888.19
5    Women's Street Footwear       45095826.81
0              Men's Apparel       44763030.33
4  Women's Athletic Footwear       38975784.94

## <b><span style='color:#df2c14'> </span><span style='color:#ff6242'>Men's Street Footwear,Women's Apparel,Men's Athletic Footwear are the products that are doing well. Men's Street Footwear bringing $208,826,244 in Sales.   	</span></b>

## <b><span style='color:#df2c14'> </span><span style='color:#ff6242'>Women's Street Footwear,Women's Athletic Footwear,Men's Apparel are the products that have opportunity to improvise their presence as can be seen by units sold and Operating profit data	</span></b>

## <b><span style='color:#df2c14'> </span><span style='color:#ff6242'>Regional Market Analysis: Which stores are experiencing strong sales, and which ones are lagging?</span></b>

In [19]:
Stores_Sales = df.groupby(by='Retailer')['Total Sales'].sum().reset_index(name='Total_Sales').sort_values(by='Total_Sales', ascending=False)

In [20]:
Stores_Sales

Retailer  Total_Sales
5      West Gear  242964333.0
1    Foot Locker  220094720.0
3  Sports Direct  182470997.0
2         Kohl's  102114753.0
0         Amazon   77698912.0
4        Walmart   74558410.0

In [21]:
Stores_Sold = df.groupby(by='Retailer')['Units Sold'].sum().reset_index(name='Units_Sold').sort_values(by='Units_Sold', ascending=False)

In [22]:
Stores_Sold

Retailer  Units_Sold
5      West Gear      625262
1    Foot Locker      604369
3  Sports Direct      557640
2         Kohl's      287375
4        Walmart      206225
0         Amazon      197990